src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -16,31 +17,29 @@ public class Line extends BaseEntity {
     private String name;
     @Column
     private String color;
-    @Column
-    private long distance;
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "up_station_id", nullable = false)
-    private Station upStation;
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "down_station_id", nullable = false)
-    private Station downStation;
+    @Embedded
+    private Sections sections = new Sections();
 
-    public Line() {
+    protected Line() {
     }
 
-    public Line(String name, String color, long distance, Station upStation, Station downStation) {
-        validateLine(name, upStation, downStation);
+    public Line(String name, String color) {
+        validateLine(name);
+        validateColor(color);
         this.name = name;
         this.color = color;
-        this.distance = distance;
-        this.upStation = upStation;
-        this.downStation = downStation;
-        this.distance = distance;
     }
 
-    private void validateLine(String name, Station upStation, Station downStation) {
-        if (name == null || name.isEmpty()) throw new LineException(EMPTY_LINE_NAME.getMessage());
-        if (upStation == null || downStation == null) throw new LineException(EMPTY_STATION.getMessage());
+    private void validateLine(String name) {
+        if (name == null || name.isEmpty()) {
+            throw new LineException(EMPTY_LINE_NAME.getMessage());
+        }
+    }
+
+    private void validateColor(String color) {
+        if (color == null || color.isEmpty()) {
```

Comment:
```suggestion
        if (Objects.isNull(color) || color.isEmpty()) {
```
중요한 것은 아니지만 이렇게 체크해줄 수도 있을 것 같습니다 ㅎㅎ